Assets/Scripts/UI/InGameUI/MenuLeft/ConstructionMenu.cs

```diff
@@ -50,6 +50,24 @@ public void RebuildMenuButtons(bool showAllFurniture = false)
         RenderFurnitureButtons();
     }
 
+    public void FilterTextChanged(string filterText)
+    {
+        Transform contentTransform = this.transform.FindChild("Scroll View").FindChild("Viewport").FindChild("Content");
+
+        List<Transform> childs = contentTransform.Cast<Transform>().ToList();
+
+        foreach (Transform child in childs)
+        {
+            Text buttonText = child.gameObject.transform.GetComponentInChildren<Text>();
+
+            string buildableName = buttonText.text;
+
+            bool nameMatchFilter = string.IsNullOrEmpty(filterText) || buildableName.ToLower().Contains(filterText.ToLower());
```

Comment:
Yep this is the desired behavior. I want to see "Cloning pod" if I search pod for instance.